File: package/main/src/Math/toKelvin.ts
import { addition } from "./addition";

/**
 * Converts temperature from Celsius to Kelvin
 * @param {number} celsius Temperature in Celsius
 * @returns {number} Temperature in Kelvin
 * @example toKelvin(26.85); // 300
 * @example toKelvin(0); // 273.15
 * @example toKelvin(-273.15); // 0 (absolute zero)
 */
export const toKelvin = (celsius: number): number => addition(celsius, 273.15);

File: package/umt_python/src/unit/to_kelvin.py
from ..math.addition import addition


def to_kelvin(celsius: float) -> float:
    """
    Converts temperature from Celsius to Kelvin.

    Args:
        celsius: Temperature in Celsius.

    Returns:
        Temperature in Kelvin.

    Example:
        >>> to_kelvin(26.85)
        300.0
        >>> to_kelvin(0)
        273.15
        >>> to_kelvin(-273.15)
        0.0
    """
    return addition(celsius, 273.15)
